build.gradle

```diff
@@ -175,8 +175,13 @@ subprojects {
             cacheChangingModulesFor 5, 'days'
         }
 
-        exclude(group: "ch.qos.logback", module: "logback-core")
-        exclude(group: "ch.qos.logback", module: "logback-classic")
+        if (!project.name.contains('cassandra-ticket-registry')) {
```

Comment:
All of this should really go away. I understand there are issues with logback requirements, and such and it's fine we have to make exceptions in the build to bypass those, but all changes need to preferably be encapsulated inside the project's own build file. Not outside.